tests/bindings/javascript/importsource.test.js

```diff
@@ -13,3 +13,43 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
+
+const loadLibCellML = require('libcellml.js/libcellml.common')
+let libcellml = null
+
+describe("Import Source tests", () => {
+    beforeAll(async () => {
+        libcellml = await loadLibCellML()
+    })
+    test('Checking ImportSource url manipulation.', () => {
+        const iS = new libcellml.ImportSource()
+
+        expect(iS.url()).toBe("")
+
+        iS.setUrl("url_string")
+
+        expect(iS.url()).toBe("url_string")
+    })
+    test('Checking ImportSource model manipulation.', () => {
+        const iS = new libcellml.ImportSource()
+        const m = new libcellml.Model()
+
+        expect(iS.model()).toBe(null)
+
+        iS.setModel(m)
+
+        expect(iS.hasModel()).toBe(true)
+        iS.removeModel()
```

Comment:
Minor: empty line before and after (if we want to be consistent).

Reply:
a95fe3c